Amethyst/Managers/ScreenManager.swift

```diff
@@ -93,8 +93,12 @@ final class ScreenManager<Window: WindowType>: NSObject {
     }
 
     func setNeedsReflowWithWindowChange(_ windowChange: Change<Window>) {
-        if case let .add(window) = windowChange {
+        switch windowChange {
```

Comment:
You still need to handle the `add` event because moving between screens does not necessarily entail a focus change.